src/People.js

```diff
@@ -0,0 +1,88 @@
+import React, { Component } from "react";
+import axios from "axios";
+
+export default class People extends Component {
+  constructor() {
+    super();
+    this.state = {
+      people: [],
+      info: {},
+      error: false,
+      input: "",
+      age: "",
+      name: "",
+      gender: "",
+    };
+  }
+
+  componentDidMount() {
+    this.people();
+  }
+
+  getInfo = (e) => {
+    const { value } = e.target;
+    this.setState({
+      input: value,
+    });
+  };
+
+  people = async () => {
+    try {
+      const { data } = await axios.get(`https://ghibliapi.herokuapp.com/people/`);
+      this.setState({
+        people: data,
+      });
+    } catch {
+      this.setState({
+        error: true,
+      });
+    }
+  };
+
+  getPeople = async (e) => {
+    e.preventDefault();
+    const { people, input } = this.state;
+    const person = people.filter((person) => person.name === input);
+
+    if (person[0]) {
+      this.setState({
+        name: person[0].name,
```

Comment:
this goes with the comment above:

Here you can have a person state and set it to person[0]